### PR TITLE
Correct mix format error messages

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: lib/mix/lib/mix/tasks/format.ex [hunk 1]
```diff
@@ -480,18 +480,18 @@ defmodule Mix.Tasks.Format do
   defp check!({_exits, [_ | _] = not_equivalent, _not_formatted}) do
     Mix.raise("""
     mix format failed due to --check-equivalent.
-    The following files were not equivalent:
+    The following files are not equivalent:
 
     #{to_bullet_list(not_equivalent)}
 
-    Please report this bug with the input files at github.com/elixir-lang/elixir/issues
+    Please report this bug with the input files at httsp://github.com/elixir-lang/elixir/issues
```
Comment (pdgonzalez872) [inline lib/mix/lib/mix/tasks/format.ex:487]: tiny typo here: `httsp`

Comment (josevalim) [inline lib/mix/lib/mix/tasks/format.ex:487]:
```suggestion
    Please report this bug with the input files at https://github.com/elixir-lang/elixir/issues
```

Comment (eksperimental) [reply lib/mix/lib/mix/tasks/format.ex]: ooppppssss. fixed. thank you

Comment (eksperimental) [reply lib/mix/lib/mix/tasks/format.ex]: fixed.

**Comment by fertapric** [comment]
Thanks @eksperimental!

**Pull request merged**